src/main/resources/static/js/modal.js

```diff
@@ -10,30 +10,38 @@ let modalGender = document.getElementById("modalGender");
 let modalPoints = document.getElementById("modalPoints");
 let modalPhoto = document.getElementById("modalPhoto");
 
-function showModal(id) {
-   fetch('/getPlayerDto?id=' + id)
-       .then(response => response.json())
-       .then(function (json) {
-          let player = JSON.parse(JSON.stringify(json));
-          modalTitle.textContent = player.fullName;
-          modalAge.textContent = player.age;
-          modalGender.textContent = player.gender;
-          modalPoints.textContent = player.rankingPoints;
-          if(player.photoUrl == null || player.photoUrl === ''){
-             if(player.gender === 'MALE'){
-                modalPhoto.setAttribute('src', "/img/silhouette-male.png")
-             }
-             if(player.gender === 'FEMALE'){
-                modalPhoto.setAttribute('src', "/img/silhouette-female.png")
-             }
-          } else {
-             modalPhoto.setAttribute('src', player.photoUrl);
-          }
-       })
-
+function showPlayerModalByPlayerId(id) {
+   fetchDataForPlayerModal('/getPlayerDto?id=' + id);
```

Comment:
Jakby chcieć, żeby modal pojawiał się od razu z danymi, to można by tutaj zrobić async await.